components/event-simulator/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/internal/CarbonEventSimulator.java

```diff
@@ -56,6 +56,7 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.*;
```

Comment:
Fixed in commit - https://github.com/wso2/carbon-event-processing/pull/267/commits/d164420dea07fda6eac29b50e03ed37a3201598d